2-mandatory/3-financial-times.js

```diff
@@ -5,16 +5,29 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  // TODO
+  let articleArr = [];
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (allArticleTitles[i].length <= 65) {
+      articleArr.push(allArticleTitles[i]);
+    }
+  }
+  return articleArr;
 }
-
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
     // TODO
+   let wordsCount= [];
+    for (let i = 0; i < allArticleTitles.length; i++){
+      wordArticle = allArticleTitles[i].split("");
+      wordsCount.push(wordArticle.length);
+    }
+    let index = wordsCount.indexOf(Math.min(...wordsCount))
```

Comment:
Wow nice use of the spread operator `...` to convert an array into a list of arguments 👏